test/unit/math/prim/mat/meta/is_constant_struct_test.cpp

```diff
@@ -28,4 +28,10 @@ TEST(MetaTraits, isConstantStruct) {
   EXPECT_TRUE(is_constant_struct<const_v1>::value);
   EXPECT_TRUE(is_constant_struct<const_v2>::value);
   EXPECT_TRUE(is_constant_struct<const_v3>::value);
+  bool temp = is_constant_struct<const_t1, const_t2>::value;
```

Comment:
I'd rather not see temporaries used like this and redefined. I'd suggest one of
1. just defining the tests directly,
2. using a new variable per test, or
3. define a helper test function.

Reply:
Unfortunately `EXPECT_TRUE(is_constant_struct<const_t1, const_t2>::value);` doesnt compile so I had to resort to this "hack". I will try (3) if I can find an easy solution, or will go with 2.